### ISSUE TITLE
uppercase OCR model

### ISSUE THREAD

**Issue by atadria** [NVIDIA-AI-IOT/NVIDIA-Optical-Character-Detection-and-Recognition-Solution#18]
Hi, with OCRNet - deployable_v2.1 model (https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/ocrnet/version ) I get lowercase text. Character list contains only lowercase letters. Is there any model with uppercase and lowercase letters?

**Comment by Tyler-D** [comment]
No. We don't have pretrained models for both uppercase and lowercase letters.

But it should be straightforward to train a model with more classes.
